Reportedly/AppDelegate.swift

```diff
@@ -20,6 +21,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Showing the launch screen 0.5s more, right now it is dismissing to fast
         Thread.sleep(forTimeInterval: 0.5)
         
+        // Request for notifications permission when first open app
+        let options: UNAuthorizationOptions = [.alert, .sound, .badge]
+        UNUserNotificationCenter.current().requestAuthorization(options: options) {
+            (didAllow, error) in
```

Comment:
Could we use `isAuthorised` or something like that instead of `didAllow `? 😄

Reply:
oh this naming is from Apple's Documentations 😅 but I think it is okay to use your name, can update 👍